.github/workflows/build.yml

```diff
@@ -26,7 +26,16 @@ jobs:
           chmod +x deps/bin/jq
           echo "${PWD}/deps/bin" >> $GITHUB_PATH
       - name: Test
+        env:
+          TEST_COVERAGE: 1
         run: make test
+      - name: Upload Coverage
```

Comment:
Is this the step that will fail the build if the code coverage is not high enough? or is that configured in codecov and added as a different GitHub check?

Reply:
I **think** so... checking...

Reply:
So it won't actually fail `test-linux` or `test-windows` or `build` - it will show up as a separate "job" that's failing: https://github.com/natalieparellano/lifecycle/pull/2